int32 node_id
string namespace

---
bool success
string msg
